tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -170,8 +171,8 @@ def config_mirror(self, peer_cluster, **kw):
         secondary_cluster = "secondary"
```

Comment:
[Optional] : Can we please add doc to this method ?

Reply:
Done.